lib/diaspora_federation/entities/relayable.rb

```diff
@@ -103,6 +99,17 @@ def to_json
         }
       end
 
+      # The order for signing
+      # @return [Array]
+      def signature_order
+        if @signature_order
```

Comment:
This doesn't work, because the entities are frozen after creation, but I used `||` instead of `||=` for now. I think I will simply always set `@signature_order` in the constructor (so set the default order there too) after the hack for comments is gone, but for now I think that is OK.